model/Core/Classes/SupportRelationship.md

```diff
@@ -0,0 +1,37 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SupportRelationship
+
+## Summary
+
+Describes how an Agent Provides Support for an Element
```

Comment:
Should we have the "to" of the relationship be an `Artifact` rather than an `Element`?  It would seem that only artifacts would be something that would require support.